cmd/ingress-operator/start.go

```diff
@@ -32,7 +32,9 @@ import (
 const (
 	// defaultTrustedCABundle is the fully qualified path of the trusted CA bundle
 	// that is mounted from configmap openshift-ingress-operator/trusted-ca.
-	defaultTrustedCABundle = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+	defaultTrustedCABundle           = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+	defaultGatewayAPIOperatorChannel = "stable"
+	defaultGatewayAPIOperatorVersion = "servicemeshoperator.v2.6.2"
```

Comment:
Isn't it:

```suggestion
	defaultGatewayAPIOperatorVersion = "servicemeshoperator.v3.0.0"
```

(or am I misunderstanding)?

Reply:
Yeah, this PR doesn't include any changes for OSSM 3.0 support, so if we merged it before #1152, it would break things. The subscription management logic will be the same, but we'll need to change the package, channel, and version strings.